packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,352 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
+    <VbCard title="Default">
+      <va-tree-view
+        :nodes="nodes"
+        node-key="id"
+      />
+    </VbCard>
+    <VbCard title="Icons">
+      <va-tree-view
+        :nodes="nodesWithIcons"
+        node-key="id"
+        expand-all
+      >
+        <template #icon="{icon}">
+          <va-icon
+            v-if="icon"
+            :name="icon"
+            size="20px"
+          />
+        </template>
+      </va-tree-view>
+    </VbCard>
+    <VbCard title="Selection and icons">
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <br />
+      <p>
+        {{ JSON.stringify(selectedNodes) }}
+      </p>
+      <br />
+      <va-tree-view
+        v-model="selectedNodes"
+        :nodes="nodesWithIcons"
+        node-key="id"
+        expand-all
+        selectable
```

Comment:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/5394573/183222868-9e80e274-4416-4f30-894b-9f63b071e4ab.png">

When some items in category are selected - I think we should make it indeterminate.